lib/register/index.js

```diff
@@ -0,0 +1,33 @@
+import check from '../check';
+import { code, message } from '../consts';
+
+/**
+ * Self register to window load
+ * Throw the error globally
+ */
+export default function register() {
+    function asynchronous() {
```

Comment:
This function could be defined outside of the main export for better readability.